crates/solidity/outputs/npm/package/src/generated/index.d.ts

```diff
@@ -795,6 +796,19 @@ export namespace parse_output {
     createTreeCursor(): cursor.Cursor;
   }
 }
+export namespace query {
+  export interface QueryResult {
+    queryNumber: number;
+    bindings: { [key: string]: cursor.Cursor[] };
+  }
+  export class Query {
+    static parse(text: string): Query;
+    get text(): string;
```

Comment:
Since `text()` can return different strings than the passed in `parse()`, I wonder if we should even expose it at all? what is the use case here?

If it is is needed on the user side, I wonder if we should use something like `stringify()` then.

Reply:
YAGNI for now